{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -136,6 +136,12 @@ matrix:
         #   stage: Initial tests
         #   env: TOXENV=codestyle
 
+        - os: linux
+          python: 3.8
+          name: RST link check
+          stage: Comprehensive tests
```

Comment:
I see little value in running link check for every push event. Perhaps cron? Also some services block CI, so if you want to add this in the template, you also need to add the code on how to skip links, set up custom timeouts, etc for it.

Reply:
I agree that every push event can be excessive. However, this repository does not include a cron template. I don't have experience setting those up...

As for the link skipping and custom timeouts, I'll turn this into a draft PR while I add some stuff.

Reply:
I've moved it to a cron, using the examples below.